src/common/enums/navigation/navigationScreen.ts

```diff
@@ -1,6 +1,9 @@
 enum NavigationScreen {
   HOME = 'Home',
-  SIGN_IN = 'SignIn'
+  SIGN_IN = 'SignIn',
+  TABS_NAV = 'TabsNav',
```

Comment:
This is not exactly screen, it's a navigation.

Reply:
what do you offer

Reply:
This is a screen name, so NavigationScreenName could fit in